lib/src/diff/main.dart

```diff
@@ -269,15 +269,15 @@ List<Diff> _diffLineMode(String text1, String text2, double timeout,
  * Returns a List of Diff objects.
  */
 List<Diff> diffBisect(String text1, String text2, double timeout,
-                      DateTime deadline) {
+                      DateTime? deadline) {
   // Cache the text lengths to prevent multiple calls.
   final text1_length = text1.length;
   final text2_length = text2.length;
   final max_d = (text1_length + text2_length + 1) ~/ 2;
   final v_offset = max_d;
   final v_length = 2 * max_d;
-  final v1 = new List<int>(v_length);
-  final v2 = new List<int>(v_length);
+  final v1 = new List<int?>.filled(v_length, 0);
+  final v2 = new List<int?>.filled(v_length, 0);
```

Comment:
I am not sure with this one.
I think there is no null value in this list. so i think List<int> should do the job